lib/autoyast.pm

```diff
@@ -523,11 +523,15 @@ EOF
 sub test_ayp_url {
     my $ayp_url = get_var('AUTOYAST');
     if ($ayp_url =~ /^http/) {
+        # replace default qemu gateway by loopback
+        $ayp_url =~ s/\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}/127.0.0.1/;
+
         if (head($ayp_url)) {
             record_info("ayp url ok", "Autoyast profile url $ayp_url is reachable");
```

Comment:
Let's change the message so that it's more clear that profile is reachable from the worker, so it's clear how this check is done.